modules/product/src/Form/AddToCartForm.php

```diff
@@ -234,6 +253,38 @@ protected function variationElement(array $form, FormStateInterface $form_state)
   }
 
   /**
+   * Builds the form elements for the variation type's line item.
+   *
+   * @param array $form
+   *   An associative array containing the structure of the form.
+   * @param \Drupal\Core\Form\FormStateInterface $form_state
+   *   The form state.
+   *
+   * @return array
+   *   The modified form.
+   */
+  protected function lineItemElement(array $form, FormStateInterface $form_state) {
+    /** @var \Drupal\commerce_product\Entity\Product $product */
+    $product = $form['#product'];
+
+    $form_display = $this->getLineItemFormDisplay($product);
+
+    $line_item_stub = LineItem::create([
+      'type' => $form_display->getTargetBundle(),
+    ]);
+
+    $form_display->buildForm($line_item_stub, $form, $form_state);
+
+    if (isset($form['quantity'])) {
```

Comment:
We need to remove all quantity settings. The default quantity can be set in the field settings, we don't need to care about it.